src/DependencyInjection/Configuration.php

```diff
@@ -184,6 +184,11 @@ private function createPersistenceNode(): NodeDefinition
                             ->cannotBeEmpty()
                             ->defaultValue('default')
                         ->end()
+                        ->scalarNode('table_prefix')
```

Comment:
The reference configuration file in the documentation (https://github.com/thephpleague/oauth2-server-bundle/blob/v0.4.1/docs/index.md#installation) needs to be updated with this option too.

Reply:
Fixed